examples/https.rs

```diff
@@ -10,6 +10,19 @@ fn main() {
     let ssl = Openssl::with_cert_and_key("examples/assets/self_signed.crt", "examples/assets/key.pem").unwrap();
     let mut server = Nickel::new();
     server.get("**", middleware!("Hello World from HTTPS"));
+    
+    /*
+     * Hint: If you want to use the `listen_https` function in your own crate / module / project, you need to
+     * add `nickel/ssl` to the features section of your crate configuration, to get nickel compiled
+     * with ssl feature activated:
+     * 
+     * Cargo.toml >
+     *      [features]
+     *      ssl = ["nickel/ssl"]
+     *
+     * As you can see, to append `hyper/ssl` in the features section is obsolete, as hyper will get compiled with
+     * ssl feature activated via `nickel/ssl`.
+    */
```

Comment:
I'd expect more people to use the form:
```toml
[dependencies]
nickel = { version = "0.8", features = ["ssl"] }
```

Adding a feature to the crate is a very heavyweight way to do this (I just advised the minimal amount of change to your toml in the other issue).

I'm happy to take a comment like this, but I think it can be much shorter, something like: 
```rust
// make sure to activate the "ssl" feature for this method to be available by adding the following to
// your Cargo.toml file.
// ```toml
// [dependencies]
// nickel = { .... }
// ```
```
additional nit: `//` per line is preferred over `/* .. */`